doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -688,8 +688,6 @@
     "        # redirect logs to relative path instead of default ~/ray_results/\n",
     "        storage_path=\"my_Tune_logs\",\n",
     "        name=\"batch_tuning\",\n",
-    "        # Set Ray Tune verbosity. Print summary table only with levels 2 or 3.\n",
```

Comment:
Do we have a timeline to enable new output by default? Currently the RunConfig docstring is a little confusing.

```python
verbose – 0, 1, 2, or 3. Verbosity mode. 0 = silent, 1 = only status updates, 
2 = status and brief results, 3 = status and detailed results. Defaults to 3. If the 
RAY_AIR_NEW_OUTPUT=1 environment variable is set, uses the new context-aware 
verbosity settings: 0 = silent, 1 = default, 2 = verbose.
```

Not needed for now, but would be great to have examples comparing outputs in verbosity level 0, 1, 2

Reply:
We aim to enable this per default in 2.7